index.ts

```diff
@@ -359,8 +398,20 @@ export class BalancerPoolsAPI extends Stack {
     });
     addCorsOptions(checkWallet);
 
-    /** 
-     * Web Application Firewall 
+    const tenderlySimulate = api.root.addResource('simulate');
+    tenderlySimulate.addMethod('POST', tenderlySimulateIntegration);
+    addCorsOptions(tenderlySimulate);
+
+    const tenderlyContracts = api.root.addResource('contracts');
+    tenderlyContracts.addMethod('POST');
```

Comment:
What is this contracts endpoint supposed to do? It doesn't have a lambda linked to it